spec/compiler/codegen/sizeof_spec.cr

```diff
@@ -243,4 +243,105 @@ describe "Code gen: sizeof" do
 
       z)).to_i.should eq(16)
   end
+
+  describe "alignof" do
```

Comment:
**nitpick:** The hierarchical position doesn't seem right with `alignof` and `instance_aligneof` positioned inside `sizeof`. They should be on the same level.